### PR TITLE
Update Foundation.md

### PR DESCRIPTION
**Pull request by Sander3003**
LFenergy foundation added

### CODE REVIEW

**Comment by CLAAdmin** [comment]
[CLA assistant check](http://cla.opensmartgridplatform.org:80/OSGP/Documentation?pullRequest=217) <br/>You should sign our Contributor License Agreement in order to get your pull request merged.<br/>

File: Opensourcecommunity/Foundation.md [hunk 1]
```diff
@@ -1,13 +1,3 @@
...
-The foundation will be non-profit. Our aim is that companies / organizations can make a sponsorship contribution to the foundation, in order to support the use and development of the open source platform.
-
-Planned key activities of the foundation are:
-
-* Maintaining and developing the Open Smart Grid Platform via a publicly accessible source with an open source initiative recognized license (Apache 2.0).
-* Providing information (manuals, publications, release notes, white papers, etc.)
-* Taking care of legal conditions
-* Community management, events for users and developers
-* Marketing and promotion of the software
+Grid eXchange Fabric (GXF) is a project of [LFenergy](https://www.lfenergy.org/). LFenergy is part of the [Linux foundation](https://www.linuxfoundation.org/).
```
Comment (rlemmers) [inline Opensourcecommunity/Foundation.md]: LFenergy => LF Energy (occurs in two places)

**Comment by Sander3003** [comment]
I tried to address your suggestions. However I think that foundation is still relevant. What is in the name: Linux Foundation. More work needs to be done to rename the documentation and Website.

**Comment by Sander3003** [comment]
Thanks for your input @rlemmers

**Pull request merged**